mobile-app/App.tsx

```diff
@@ -1,19 +1,39 @@
-import { StatusBar } from 'expo-status-bar';
-import { StyleSheet, Text, View } from 'react-native';
+import * as React from 'react';
+import { StyleSheet, View, ScrollView } from 'react-native';
+import Board from './components/Board';
+import { GameState } from '../web-app/src/common/gameState';
```

Comment:
Is it ok to use a component from another app ?

Consider copy paste it in your "mobile-app" folder, this folder should works in standalone

Reply:
Florian gave us a solution for this(sharing all the common stuff with a local package and npm link) but I haven't tried it yet ! Does it work for you?

Reply:
Yep it's very complicated just for factorize `GameState` 

I think you can just copy paste `GameState` inside `mobile-app` folder and it should be good :)

Reply:
It's just a little bit ugly to import something from another project, that's the main point here

Reply:
Ok will make some duplication for now and probably use the local package idea later when we will really need it.